rest_framework_json_api/renderers.py

```diff
@@ -643,6 +643,21 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
         else:
             render_data['data'] = json_api_data
 
+        if included_cache:
+            if isinstance(json_api_data, list):
+                objects = json_api_data
+            else:
+                objects = [json_api_data]
+
+            for object in objects:
+                obj_type = object.get('type')
+                obj_id = object.get('id')
+                if obj_type in included_cache.keys() and \
```

Comment:
Could be simplified to:
```suggestion
                if obj_type in included_cache and \
```

Reply:
Thanks for pointing this out. I some missed this one.